web/gds-user-ui/src/modules/dashboard/organization/organizationService.ts

```diff
@@ -1,7 +1,10 @@
 import axiosInstance from 'utils/axios';
-
-export const GetAllOrganisations = async () => {
-  const response = await axiosInstance.get(`/organizations`);
+export const getAllOrganisations = async (page?: number) => {
+  const currentPage = page || 1;
```

Comment:
You don't need here to add a default number.

https://github.com/ryanmcdermott/clean-code-javascript#use-default-parameters-instead-of-short-circuiting-or-conditionals